vignettes/site_only/db_connections.Rmd

```diff
@@ -0,0 +1,183 @@
+---
+title: "Database Connections using healthcare.ai"
```

Comment:
Let's make the title "Database Connections" since everything on the site will be healthcare.ai

Reply:
Done.